src/random_number.py

```diff
@@ -4,3 +4,8 @@
 #   Output:
 #   The random number is: 4
 #
+from random import randint
+RAND_RANGE = 986234
+
+rand_number = randint(0,RAND_RANGE)
+print(rand_number)
```

Comment:
you could wrap everything into a function
moreover, you should use `if __name__ == __main__`  strategy https://www.programmareinpython.it/video-corso-python-intermedio/03-if-name-main/